docs/build-on-linea/gas-fees.mdx

```diff
@@ -1,139 +1,108 @@
 ---
-title: Gas fees on Linea
+description: How to estimate Linea gas costs
 sidebar_position: 6
 ---
 
-If you have interacted with the Ethereum ecosystem, then you've likely encountered the unpredictability of gas fees, which can swing widely based on network activity. 
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
 
-For a refresher on gas, click [here](https://support.metamask.io/hc/en-us/articles/4404600179227-User-Guide-Gas#:~:text=A%20normal%20transaction%20sending%20ETH,transactions%20also%20cost%2021%2C000%20gas.). 
+# Estimate transaction costs
 
-As a Layer 2 blockchain, Linea provides a more stable and cost-effective solution for transaction fees. 
+Linea supports the [Ethereum EIP-1559 gas price model](https://ethereum.org/developers/docs/gas); however, being a
+layer 2 blockchain, Linea provides a more stable and cost-effective solution for transaction fees.
```

Comment:
```suggestion
layer 2 blockchain, Linea gas API provides a more accurate and cost-effective solution for transaction fees.
```

Reply:
This intro is trying to explain that the Linea network provides a more cost effective and stable solution to Ethereum (being a L2). I haven't yet introduces the API. However I'll be sure to highlight "accuracy" when I introduce the API.